docker/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2019 Jonathan VUILLEMIN
```

Comment:
Is this ok?

Reply:
@jbout should be fixed into develop later